projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1689,7 +1689,7 @@ public static function wrap_message_in_html_tags( $title, $body, $footer ) {
 				'',
 				$template
 			),
-			$title,
+			( ! empty( $title ) ? '<h1>' . $title . '</h1>' : '' ),
```

Comment:
No need for `empty` since it's always going to be set. Also having `! empty() ? ... : ...` is really kind of confusing.

As for the replacement, I'm most inclined to do like this
```suggestion
			( $title !== '' ? '<h1>' . $title . '</h1>' : '' ),
```
but then in case of people doing `__return_false` or something incorrect like that for the filter we may want to add a string cast on line 1402.

Reply:
👍 Done in b8021145bd1a638e430306fa01200b2b03fdc6e7 and 6cd95724451f749541d89f44049f659f56c87502